courses/free-courses-en.md

```diff
@@ -650,10 +650,12 @@
 ### DevOps
 
 * [AWS DevOps Engineer Learning Plan](https://explore.skillbuilder.aws/learn/public/learning_plan/view/85/devops-engineering-learning-plan?la=sec&sec=lp) - AWS
+* [Build and Deploy a Production-Ready API | Full DevOps Course 2025](https://youtu.be/H5FAxTBuNM8?si=iJRmX8k2soVH1_KK) - JavaScript Mastery
```

Comment:
For youtube, we prefer "playlist" links and the name of the producer/creator. docs/CONTRIBUTING.md#platforms-and-access-notes